model/tab/gear/armor-main.html

```diff
@@ -0,0 +1,31 @@
+<div class="main border">
+    <div class="weight">
+        <label>{{localize "GEAR.WEIGHT"}}</label>
+        <select name="data.weight">
+            {{#select data.weight}}
+                <option value="none">{{localize "WEIGHT.EQUIPPED"}}</option>
+                <option value="tiny">{{localize "WEIGHT.TINY"}}</option>
+                <option value="light">{{localize "WEIGHT.LIGHT"}}</option>
+                <option value="regular">{{localize "WEIGHT.REGULAR"}}</option>
+                <option value="heavy">{{localize "WEIGHT.HEAVY"}}</option>
+            {{/select}}
+        </select>
+    </div>
+    <div class="part">
+        <label>{{localize "ARMOR.PART"}}</label>
+        <select name="data.part">
+            {{#select data.part}}
+                <option value="body">{{localize "ARMOR.BODY"}}</option>
+                <option value="head">{{localize "ARMOR.HELMET"}}</option>
+                <option value="shield">{{localize "ARMOR.SHIELD"}}</option>
+            {{/select}}
+        </select>
+    </div>
+    <div class="features">
+        <label>{{localize "WEAPON.FEATURE"}}</label>
+        <textarea name="data.features">{{data.features}}</textarea>
+    </div>
+</div>
+<div class="roll-modifiers border">
+    {{> systems/forbidden-lands/model/partial/roll-modifiers.html }}
```

Comment:
Tabulated partial inclusion (although it doesn't really matter for `roll-modifiers.html`).

Reply:
Thanks! I will fix this:)

Reply:
On a separate note. I am eager to push a dev env refactor after this pull. I'll be renaming all the templates `.hbs` for correct syntax highlighting in IDEs, I'll also be adding a prettier formatting config that prevents these partials references from being formatted wrongly.